### PR TITLE
Add more CSS customization and CSS Themes

### PR DESCRIPTION
**Pull request by prayag17**
I have added customization for the Icon Pack, have also fixed the CSS links, and added more known CSS themes.

### CODE REVIEW

File: general/clients/css-customization.md [hunk 1]
```diff
@@ -555,6 +555,39 @@ An example of a color theme.
 }
 ```
 
+### Change Icon Pack
+You can choose between Material Icons(Icon Pack used by Jellyfin) and Fontawesome icons.
```
Comment (Shadowghost) [inline general/clients/css-customization.md:559]:
```suggestion
You can choose between Material Icons (Icon Pack used by Jellyfin) and Fontawesome icons.
```

File: general/clients/css-customization.md [hunk 2]
```diff
@@ -569,11 +602,15 @@ Keep in mind that these posts may have been made under previous versions of Jell
...
 
 * [Monochromic - A custom theme for Jellyfin mediaserver created using CSS overrides](https://github.com/CTalvio/Monochromic)
 * [Kaleidochromic - Yet another custom theme for Jellyfin mediaserver created using CSS overrides, built on top of Monochromic](https://github.com/CTalvio/Kaleidochromic)
 * [Novachromic - A light theme, built on top of Monochromic](https://github.com/CTalvio/Novachromic)
-* [JellyfinCSS - This is Jellyfin custom-css](https://github.com/prayag17/JellyfinCSS)
+* [JellySkin - Vibrant Jellyfin theme with a lot a animations](https://github.com/prayag17/JellySkin)
+* [JellyFlix - The Best Netflix Clone for Jellyfin](https://github.com/prayag17/JellyFlix)
 * [Jellyfin Netflix Dark - The Best Netflix Dark Theme for Jellyfin Around!](https://github.com/DevilsDesigns/Jellyfin-Netflix-Dark)
+* [Dark and Green - A Emby like night mode skin](https://github.com/mbcooper83/jellyfin-css-darkandgreen)
+* [HInt Of Green](https://github.com/looi-wh/HintOfGreenCSS)
```
Comment (Shadowghost) [inline general/clients/css-customization.md:616]:
```suggestion
* [Hint of Green](https://github.com/looi-wh/HintOfGreenCSS)
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
There's a bunch of (valid) linter warnings and you didn't fix the feedback from @Shadowghost (but you marked it as resolved).

**Comment by prayag17** [comment]
I have fixed the linter warnings and have made all the requested changes

**Review by Shadowghost — APPROVED** [review]

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**